src/routes/__root.tsx

```diff
@@ -0,0 +1,19 @@
+import { createRootRoute, Outlet } from '@tanstack/react-router'
+import { TanStackRouterDevtools } from '@tanstack/router-devtools'
+
+export const Route = createRootRoute({
+    component: () => (
+        <>
+            <div className="p-2 flex gap-2">
+                {/* <Link to="/" className="">
+                    Index
+                </Link>{' '}
+                <Link to="/manga" className="">
+                    Manga
+                </Link> */}
```

Comment:
If you are planning to use this code in the future, please add a comment indicating so, for example `For future use` before the code that has been commented out. If you are no longer using or planning to use the code, please remove it.